app/routes.py

```diff
@@ -1 +1,237 @@
-from flask import Blueprint
\ No newline at end of file
+from flask import Blueprint, jsonify, request, abort, make_response
+from app import db
+from app.models.task import Task
+from app.models.goal import Goal
+from datetime import datetime
+import os, requests
+
+tasks_bp = Blueprint("tasks", __name__, url_prefix="/tasks")
+goals_bp = Blueprint("goals", __name__, url_prefix="/goals")
+
+#***********************************************#
+#***************** Task Routes *****************#
+#***********************************************#
```

Comment:
The routes here are very cleanly written, making great use of helper functions.  Well done.